### DISCUSSION TITLE
Displaying "cards"

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#801] [Ideas]
When selecting "view all" only the first 3 available cards are presented in the top row of 4(so it seems) positions . "Position 4" is empty and therefore for me it now looks like there are no more cards available.  Only by "scrolling down" the other cards become visable.

Example:
![afbeelding](https://user-images.githubusercontent.com/102361398/190432933-21e09724-b649-43f9-ab50-8912898d63a5.png)

**Comment by techsmyth** [comment]
Hi Gerlof, apologies for the slower reply...

This has now been raised on the development backlog as a bug: https://github.com/alkem-io/client-web/issues/2707 - hopefully it will make the sprint starting Oct 10th.

On a side note, I see that the language in the callout is still the default english text, it may be useful to edit that to be Dutch?

Mvg, Neil

**Reply by GerlofdeHaan** [reply]
Hi Neil, thanks for the update. 
In regards to your side note: I noticed it too, but I assumed that it would be "tackled" in de translation update. Or did you expect the administrator to edit (and in that way translate) the text in explore section (call outs, contribution, en collaboration). 
If so, no problem, than I will do that.

**Reply by techsmyth** [reply]
Hi Gerlof, 

The text that describes a callout is actually not part of the "translations", but actually content that is specific to the hub. 

In migrating to the usage of Callouts we could not automatically detect the language as a Hub does not actually at the moment have a formal language - so this text was added in english. It would be best if you customize it to Dutch, and of course ideally make it personal to the T3D hub!

Cheers, Neil

**Reply by GerlofdeHaan** [reply]
Ok! thanks.
